plugins/module_utils/proxmox_interfaces.py

```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Andreas Botzner (@botzner_andreas) <andreas at botzner dot com>
```

Comment:
`botzner_andreas` is not a valid GitHub username. You have to use your GitHub username here (and in all other places).